### DISCUSSION TITLE
5G solution with srsRAN gNB - question About Benetel RF RAN650

### DISCUSSION THREAD

**Discussion by ijazzix** [srsran/srsRAN_Project#220] [General Help]
Hello

My company conducting functional tests of 5G CORE (Open5GS) and gNB (RAN) modules from srsRAN. It seems that at the moment, gNB based on srsRAN is the best solution for network deployment. However, we are currently in need of a good programmable radio (RF) that can provide radio communication to a large number of network users. 
**In connection with the above, I would like to inquire about some details:**

**1.** Is the Benetel RAN650 RF compatible with the gNB included in the srsRAN project?
There is some unclear information on this in the following threads, but there is no specific answer:
https://github.com/srsran/srsRAN_Project/discussions/90
https://github.com/srsran/srsRAN_Project/discussions/91
I do know that RAN550 is compatible because the srsRAN project provides information on how to connect it, for example:
https://docs.srsran.com/projects/project/en/latest/tutorials/source/oranRU/source/index.html

**2.** We are interested in RF RAN650 compatibility with srsRAN because this RF is of the outdoor type and provides support for a large number of users, similar to commercial networks. We believe that the twin model RAN550 is too weak for service deployment.  
We need an RF that can handle a large number of users, not a laboratory device. 
Are any of the RFs used in the srsRAN project capable of meeting these expectations? 
Can you provide any advice on this?

If I may, I have two more small questions related to using srsRAN..

**3.** What are the licensing conditions for using srsRAN? Is it possible to sell, for example, a 5G product created by us with srsRAN software? In such a case, would any portion of the profit belong to your organization?

**4.**  How many maximum users (UE) can the gNB(srsRAN) to Core(Open5GS) connection handle within a 5G network? Can you provide information about the available functionalities of srsRAN gNB (related to network deployment limitations)?
Any information would be appreciated.

Regards

**Comment by andrepuschmann** [comment]
Hey @ijazzix,
the situation regarding the RAN650 is exactly like described in #90 and #91 - we don't have tested it ourselves but it should be the same as RAN550 and 650 seem to be using the same firmware.

Regarding 2) I think you should contact Benetel.

Regarding 3+4) srsRAN Project is licensed under AGPL v3 (see https://github.com/srsran/srsRAN_Project/blob/main/LICENSE) and as such can be used in commercial projects under the conditions stated in the license.
However, I understand the topic can be quite overwhelming at the beginning. If you want to learn more about the topic I suggest you contact us under sales@srs.io and we can setup a call to discuss the matter.

I am closing the issue as the RAN650 is handled in the above mentioned tickets already.

Thanks
Andre
